api/v1alpha1/storageconsumer_types.go

```diff
@@ -90,6 +90,10 @@ type ClientStatus struct {
 	// Name is the name of connected storageclient
 	// +optional
 	Name string `json:"name,omitempty"`
+
+	// UsedCapacityInGiB is the used capacity of connected storageclient
+	// +optional
+	UsedCapacityInGiB string `json:"usedCapacityInGiB,omitempty"`
```

Comment:
what resources on client side provides this value?

Reply:
The value for this will be available in status section of `ClusterResourceQuota` CR. https://docs.openshift.com/container-platform/4.15/rest_api/schedule_and_quota_apis/clusterresourcequota-quota-openshift-io-v1.html#status 
We have used a section that will display the used capacity from the allocated capacity.
Ex:
```
status:
  total:
    hard:
      requests.storage: 17Gi
    used:
      requests.storage: 15Gi

```

Reply:
I submitted this PR as a work in progress because I accidentally set the type as a string instead of an int. I'm currently working on fixing it.

Reply:
@rchikatw The name used capacity seems misleading here. `Used` represents a portion of something, but the name does not represent the "what". So when I see the term used capacity there is a question of used of what. Is it from total cluster capacity? from the quota? from ODF available storage? 

Because this is supposed to reflect the storage quota utilization, I will prefer if we will call it something like:
`StorageQoutaUtilizationInGB and maybe we should consider to report it as a ratio and not as a number

WDYT?

Reply:
I have updated the name to `StorageQuotaUtilizationInGiB`. I don't think using ratio is a good choice for displaying storage utilization in the UI. The UI would have to perform the calculation to display it in the column(In GiB). The ratio would be useful only for triggering alerts, so it would be better to keep that logic in the backend.

Reply:
Doesn't the UI need to convert that back into a ration? they are going to create a filled bar out of the value, and they already have the quota itself, so converting a ratio back into a quota is just 1 multiplication. I would even say that displaying a ratio makes more sense to me then displaying a value. We can have a discussion with the UX eng and then settle on a representation of the value.

Reply:
The bar will need to report values also. On hover we could show the actual values.

Reply:
As per our chat discussion, we have decided to store the ratio in the storage consumer status section. The UI team will determine whether to display it as a bar or in GIB based on the UI requirements.